cdap-docs/admin-manual/source/security.rst

```diff
@@ -456,30 +449,27 @@ security components are working as expected:
 
     {"auth_uri":["http://localhost:10009/token"]}
 
-- Submit a username and password to one of the URIs to obtain an ``AccessToken``:
+- Submit a username and password to one of the authentication URIs (``<auth-uri>``) to
+  obtain an ``AccessToken``::
 
-  .. tabbed-parsed-literal::
-	
-    curl -v -w"\n" -X GET -u <username>:<password> <auth-uri>
+    GET <auth-uri> <username>:<password>
```

Comment:
This should be something more like `GET <auth-uri> "Authorization: Basic abcd1234=="` with an explanation on how to create the encoded username:password pair... "or you can use curl like..." below, with a username:password pair and it'll generate the header for you.